runtimes/standalone/src/lib.rs

```diff
@@ -314,6 +314,9 @@ impl attestation::Config for Runtime {
 	type Currency = Balances;
 	type Deposit = AttestationDeposit;
 	type MaxDelegatedAttestations = MaxDelegatedAttestations;
+	type AttesterId = DidIdentifier;
+	type AuthorizationId = ();
+	type AccessControl = ();
```

Comment:
Q: What is the reason for not mirroring the parachain runtimes here?

Reply:
yeah overlooked

Reply:
[e675ecc](https://github.com/KILTprotocol/mashnet-node/pull/316/commits/e675ecc7ef7caa82c9b9810be996997b15b86717)